package.json

```diff
@@ -24,7 +24,7 @@
     "index.js"
   ],
   "peerDependencies": {
-    "eslint": "^0.15.0"
+    "eslint": "^0.16.0"
```

Comment:
`peerDependencies` shouldn't be this strict:
> https://docs.npmjs.com/files/package.json#peerdependencies

If this project works on both versions' API, then the range should be used here instead.